### ISSUE TITLE
Bot/D2 crashes when I'm away

### ISSUE THREAD

**Issue by crashingblo** [kolton/d2bot-with-kolbot#324]
Around two hours after I'm not present, bot/d2 client appears to crash. Like, if I go to my university and I leave my bot working, 2 hours later when I get back home, I won't see neither D2, neither the bot. Both crash whit no error message whatsoever and I can't seem to make it run for longer periods of time. 
Please help.

**Comment by crashingblo** [comment]
Also, I made some changes in the char config cuz I wanted him to invite players to party, and now when he enters Baal wp, he immediately leaves game. Please help.

**Comment by jmkim2ea** [comment]
Yes me too. my bot also immediately leaves game from today. I guess it is after Blizzard unknown new patch. Blizzard now ban the D2BS and move to the D2NT. Today I test  both D2BS is blocked and now D2NT is ok. 
After next ladder, D2BS should be changed, without out changed it will be useless.  I hope D2BS alive. :-)

**Comment by noah-** [comment]
Do you have a d2bs or d2bot crash log?
